### PR TITLE
adding check_queue job

### PR DESCRIPTION
**Pull request by Bharath92**
https://github.com/Shippable/beta/issues/554

### CODE REVIEW

File: check_queue.sh [hunk 1]
```diff
@@ -0,0 +1,93 @@
+#!/bin/bash -e
```
Comment (shrivara) [inline check_queue.sh]: Is this file executable?

File: shippable.resources.yml [hunk 2]
```diff
@@ -290,10 +290,39 @@ resources:
...
+
   ###############################
   # END notification resources
   ###############################
 
+  ###############################
+  # BEGIN check queue resources
+  ###############################
+
+  - name: api_params
```
Comment (rageshkrishna) [inline shippable.resources.yml]: I don't think this is a good name for this resource.

File: check_queue.sh [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+export API_PARAMS="api_params"
```
Comment (rageshkrishna) [inline check_queue.sh]: Hard-coding this makes the script too specific. For example, I can't reuse it to monitor RC queues.

File: shippable.jobs.yml [hunk 4]
```diff
@@ -1624,4 +1624,26 @@ jobs:
...
   ##########################
   # end DOCS JOBS
-  ##########################
\ No newline at end of file
+  ##########################
+
+  ##########################
+  # begin check_queue job
+  ##########################
+  - name: check_queue
```
Comment (shrivara) [inline shippable.jobs.yml]: Maybe call this `check_prod_queue`. Rest looks good to me.

**Pull request merged**